### ISSUE TITLE
New unit requests: %w/w, %w/v, %v/v

### ISSUE THREAD

**Issue by meganrwong** [qudt/qudt-public-repo#261]
w - weight
v - volume 
Concentration units used in chemistry. Appropriate to add?

**Comment by steveraysteveray** [comment]
Yes, these should be additional units related to the quantity kind:

http://qudt.org/vocab/quantitykind/VolumeFraction for the percent volume fraction unit,
and we would need to define new quantity kinds for 
percent weight fraction unit
and
percent weight density unit, or something like that...
